cxxlib.sh

```diff
@@ -4,19 +4,25 @@
 # Last edit: 29-03-2015
 
 # It creates a complete filesystem for a c++ project.
-# $1 is the name of the project
-# $2 is the name of the first class (in lower case)
+# $AUTHOR is the author of the project
+# $PROJECT_NAME is the name of the project
+# $CLASS is the name of the first class (in lower case)
 
 # Creation of the name of the class
-CLASS=$2
-CLASS=`echo ${CLASS:0:1} | tr  '[a-z]' '[A-Z]'`${CLASS:1}
-
+echo 
+read -e -p "Author: ................. " AUTHOR
+read -e -p "Name of the project: .... " PROJECT_NAME
+read -e -p "Name of the class: ...... " CLASS
+
+CLASS=`echo -e ${CLASS:0:1} | tr '[a-z]' '[A-Z]'`${CLASS:1}
```

Comment:
You have to create a new variable, e.g. CLASS_FILE_NAME, that is equal to CLASS before this line. The file name will be in lower case.